src/slack_interaction_handler.ts

```diff
@@ -112,12 +112,7 @@ const getClosedVoterPanelText = (
   // See https://api.slack.com/reference/surfaces/formatting#visual-styles
   const specialSlackTimestamp = `<!date^${timeSinceEpochSecs}^{date_num} {time_secs}|${new Date()}>`;
 
-  switch (selectedVoterStatus) {
-    case 'VOTED':
-      return `*Congratulations!* :tada: This voter was marked as *VOTED* by *${originatingSlackUserName}* on *${specialSlackTimestamp}*. On to the next one! :ballot_box_with_ballot:`;
-    default:
-      return `:no_entry_sign: This voter was marked as *${selectedVoterStatus}* by *${originatingSlackUserName}* on *${specialSlackTimestamp}*. :no_entry_sign:`;
```

Comment:
Are you removing the spam/refused notification?

Reply:
That hunk was in the other PR this was based on top of.  The change was removing the switch statement because there is no longer a 'voted' blocks that hides all the buttons; refused/spam still behave as before.